hub/powertoys/fancyzones.md

```diff
@@ -99,82 +97,80 @@ If multiple displays are in use, the editor will detect the available monitors a
 
 ### Creating a custom layout
 
-The zones editor supports creating custom layouts. Select **+ Create new layout** at the bottom.
+Select **+ Create new layout** at the bottom.
 
-There are two style of custom zone layouts: **Grid** and **Canvas**.
+There are two styles of custom zone layouts: **Grid** and **Canvas**.
 
-The **Grid** model starts with a three column grid and allows zones to be created by splitting and merging zones, resizing the gutter between zones as desired. This is a relative layout and will resize with different screen sizes. You can edit the layout using mouse or using keyboard.
+The **Grid** model starts with a three column grid and allows zones to be created by splitting and merging zones, moving the gutter between zones as desired. This is a relative layout and will resize with different screen sizes. You can edit the layout using mouse or using keyboard.
 
-#### Mouse
```

Comment:
Any particular reason you replaced the markdown headers with bold? The Learn Linter in VS Code is going to flag these as issues.

Reply:
I just felt the headers were getting a bit messy. Should I use plain text or H4?

Reply:
If they're subsections of the H3 heading, then to me it makes sense to keep them as H4 headings. It's also possible (but not very likely) that some folks have bookmarks to one of those sections. I try to only remove or change headings when absolutely necessary for this reason. If we have any other docs that link to the heading's anchor tag, it'll show up in our broken link reports.

Reply:
Can't imagine this is actually a case in practice, but I see the idea.